src/app/mock-org-cards.ts

```diff
@@ -0,0 +1,102 @@
+export interface orgCard {
+  title: string;
+  owner: string;
+  city: string;
+  ownership: string;
+  price: number | boolean;
+  rate: string; //temp
+  ageFrom: number;
+  ageTo: number;
+  category: string[];
+}
+
+export const ORGCARDS: orgCard[] = [
```

Comment:
do we still need this after mock was added?

Reply:
removed